### PR TITLE
Adding instructions for AOI instead of Lorem Ipsum

### PR DESCRIPTION
**Pull request by mhines-usgs**

### CODE REVIEW

File: dsas-web-ui/src/main/webapp/stages/shorelines.jsp [hunk 1]
```diff
@@ -18,11 +18,12 @@
 			<div id="description-aoi" class="well hidden">
 				<h3>Selecting An Area Of Interest</h3>
 				<p>
-					Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin 
-					pretium varius ipsum, sit amet sollicitudin mauris tristique non. 
-					Sed ligula leo, ullamcorper non felis sit amet, facilisis luctus dui. 
-					Aliquam vel magna nunc. Mauris quam arcu, viverra id lacus sodales, 
-					imperdiet cursus magna. In vel efficitur lacus.
+				<ul>To select an area of interest (AOI),
```
Comment (isuftin) [inline dsas-web-ui/src/main/webapp/stages/shorelines.jsp]: A <ul> tag should not have text content in it. 

You should probably put that text immediately outside the <ul> tag.

```
Text text text!
<ul> 
  <li>list item</li> 
</ul>
```

File: dsas-web-ui/src/main/webapp/stages/shorelines.jsp [hunk 2]
```diff
@@ -17,12 +17,13 @@
 			</p>
 			<div id="description-aoi" class="well hidden">
 				<h3>Selecting An Area Of Interest</h3>
-				<p>
-					Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin 
-					pretium varius ipsum, sit amet sollicitudin mauris tristique non. 
-					Sed ligula leo, ullamcorper non felis sit amet, facilisis luctus dui. 
-					Aliquam vel magna nunc. Mauris quam arcu, viverra id lacus sodales, 
-					imperdiet cursus magna. In vel efficitur lacus.
+				<p>To select an area of interest (AOI),
```
Comment (mhines-usgs) [inline dsas-web-ui/src/main/webapp/stages/shorelines.jsp]: @isuftin mas mejor?

Comment (isuftin) [inline dsas-web-ui/src/main/webapp/stages/shorelines.jsp]: According to [HTML spec](http://www.w3.org/TR/html5/dom.html#phrasing-content-1), `<p>` nodes are Phrasing Nodes and can only contain other phrasing nodes. `<ul>` is flow content and cannot be inside of phrasing nodes like `<p>`.

Could you try putting the full `<p>` node immediately above the `<ul>` node and see if this works for you?

Comment (mhines-usgs) [inline dsas-web-ui/src/main/webapp/stages/shorelines.jsp]: I can't test it to see if it works within DSAS but it seems to look OK here: 
image

@isuftin cool to merge? (I can't do that, you'll have to)

**Comment by isuftin** [comment]
:+1:

**Pull request merged**